cirq/sim/clifford/act_on_stabilizer_ch_form_args.py

```diff
@@ -0,0 +1,42 @@
+# Copyright 2020 The Cirq Developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""A protocol for implementing high performance clifford tableau evolutions
```

Comment:
I don't think we need this docstring: 1) this file contains a single class, and that is documented. 2) this is not a protocol, act_on is the protocol, this is one piece of the Clifford simulation that is implemented using the act_on the protocol.

Reply:
Makes sense. It also mentioned clifford tableau instead of StabilizerStateChForm. Thanks for catching the copy-pasta.